diffusion/models/models.py

```diff
@@ -123,6 +123,122 @@ def stable_diffusion_2(
     return model
 
 
+def stable_diffusion_xl(
+    model_name: str = 'stabilityai/stable-diffusion-2-base',
+    unet_model_name: str = 'stabilityai/stable-diffusion-xl-base-1.0',
+    vae_model_name: str = 'madebyollin/sdxl-vae-fp16-fix',
```

Comment:
Could we check if the SDXL VAE NaNs with bfloat16 within a few iteration? But there does look like weird colorization in the SDXL 1.0 VAE, so should we prefer this model?

Reply:
Will default to the `sdxl-vae-fp16-fix` VAE checkpoint for now, but will check about using the `stable-diffusion-xl-base-1.0` VAE checkpoint with bfloat16.